tests/FiberTraces.py

```diff
@@ -854,10 +794,12 @@ def suite():
 
     suites = []
     suites += unittest.makeSuite(FiberTraceTestCase)
-#    suites += unittest.makeSuite(tests.MemoryTestCase)
     return unittest.TestSuite(suites)
 
 def run(exit = False):
+    """Quiet down loggers which are too verbose"""
+    lsst.log.setLevel("CameraMapper", lsst.log.FATAL)
+    lsst.log.setLevel("afw.image.ExposureInfo", lsst.log.FATAL)
```

Comment:
Also, why did you set the level to FATAL?  Was e.g. WARN not sufficient.

Paul: what are you proposing we do now?